### PR TITLE
trezor-suite: 21.2.2 -> 21.4.1

### PR DESCRIPTION
**Pull request by ethancedwards8**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fufexan** [comment]
Result of `nixpkgs-review pr 120671` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>trezor-suite</li>
  </ul>
</details>

File: pkgs/applications/blockchains/trezor-suite/default.nix [hunk 1]
```diff
@@ -7,12 +7,12 @@
...
 let
   pname = "trezor-suite";
-  version = "21.2.2";
+  version = "21.4.1";
   name = "${pname}-${version}";
 
   src = fetchurl {
     url = "https://github.com/trezor/${pname}/releases/download/v${version}/Trezor-Suite-${version}-linux-x86_64.AppImage";
-    sha256 = "0dj3azx9jvxchrpm02w6nkcis6wlnc6df04z7xc6f66fwn6r3kkw";
+    sha256 = "khkWilBDVhUrO4B+HnKC4hlSRh0ndZbGuC3f6BrCQZw=";
```
Comment (prusnak) [inline pkgs/applications/blockchains/trezor-suite/default.nix:15]: Please don't use base64 hashes, but rather the original format.

Comment (ethancedwards8) [reply pkgs/applications/blockchains/trezor-suite/default.nix:15]: I got that hash after inserting 52 0's, building it, and replacing the 0's with the suggested hash. 

Can I use nix-prefetch-url to get the correct hash or should I use something else?

Comment (prusnak) [reply pkgs/applications/blockchains/trezor-suite/default.nix:15]: nix-prefetch-url should work just fine I guess

**Comment by prusnak** [comment]
Superseded by https://github.com/NixOS/nixpkgs/pull/120898 which also adds aarch64 support.

**Pull request closed without merging**